src/o365/aad/commands/approleassignment/approleassignment-add.spec.ts

```diff
@@ -252,7 +253,7 @@ describe(commands.APPROLEASSIGNMENT_ADD, () => {
 
     cmdInstance.action({ options: { debug: true, appId: 'fff194f1-7dce-4428-8301-1badb5518201', resource: 'SharePoint', scope: 'Sites.Read.All' } }, (err?: any) => {
       try {
-        assert.equal(JSON.stringify(err), JSON.stringify(new CommandError("The scope value 'Sites.Read.All' you have specified does not exist for SharePoint. \nAvailable scopes (application permissions) are: \nScope1\nScope2")));
+        assert.equal(JSON.stringify(err), JSON.stringify(new CommandError("The scope value 'Sites.Read.All' you have specified does not exist for SharePoint. "+ os.EOL +"Available scopes (application permissions) are: "+ os.EOL +"Scope1"+ os.EOL +"Scope2")));
```

Comment:
To make it easier to read shall we use string interpolation instead?

Reply:
Yes, good suggestion. Will update!